specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json

```diff
@@ -0,0 +1,736 @@
+              "$ref": "#/definitions/Table"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Tables"
+        ],
+        "x-ms-examples": {
+          "TablesGet": {
+            "$ref": "./examples/TablesDelete.json"
+          }
+        },
+        "operationId": "Tables_Delete",
+        "description": "Delete a Log Analytics workspace table.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TableNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition."
+          },
+          "204": {
+            "description": "OK response definition."
+          },
+          "202": {
+            "description": "Accepted response definition (Long running operation)."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Column": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Column name."
+        },
+        "type": {
+          "type": "string",
+          "description": "Column data type.",
+          "enum": [
+            "string",
+            "int",
+            "long",
+            "real",
+            "boolean",
+            "dateTime",
+            "guid",
+            "dynamic"
+          ],
+          "x-ms-enum": {
+            "name": "ColumnTypeEnum",
+            "modelAsString": true
+          }
+        },
+        "dataTypeHint": {
+          "type": "string",
+          "description": "Column data type logical hint.",
+          "enum": [
+            "uri",
+            "guid",
+            "armPath",
+            "ip"
```

Comment:
> `Values specified by 'enum' mismatch those specified by 'x-ms-enum' (name: 'ColumnDataTypeHintEnum'): 'uri', 'guid', 'armPath', 'ip' vs 'uri', 'guid', 'armPath'`

The enum value `ip` is not defined in `x-ms-enum`, please fix it.

Reply:
Done. thank you for the comment.

Reply:
One more invalid enum: `Values specified by 'enum' mismatch those specified by 'x-ms-enum' (name: 'ProvisioningStateEnum'): 'Updating', 'Succeeded' vs 'Updating', 'InProgress', 'Succeeded'`

Please check and fix the `SDK azure-resource-manager-schemas` validation.

Reply:
Done, thank you.